src/test/cli-integration/rbd/gwcli_create.t

```diff
@@ -34,7 +34,7 @@ Create the target IQN
 
 Create the first gateway
 ========================
-  $ HOST=$(python3 -c "import socket; print(socket.getfqdn())" | cut -d. -f1)
+  $ HOST=$(python3 -c "import socket; print(socket.getfqdn())" | cut -f1)
```

Comment:
IIRC the default delimiter for `cut` is a tab.  Does `socket.getfqdn()` return something with tabs in it?  If not, is `-f1` needed at all?

Reply:
socket.getfqdn() gives full domain name(hostname + domain name) I think we don't need cut at all now, will amend.